.github/workflows/test.yml

```diff
@@ -108,7 +107,7 @@ jobs:
           DD_ENV: ci
 
   summary:
-    name: Summary (Node 16)
+    name: Summary
```

Comment:
I think we'll need to update the "Required Checks" configuration on the repo since this was renamed.

Reply:
This will clash with the general "Summary" check that is required. We should call this  "Summary (Node by Version)" or something more unique like that.